include/openssl/evperr.h

```diff
@@ -10,12 +10,6 @@
 
 #ifndef OPENSSL_EVPERR_H
 # define OPENSSL_EVPERR_H
-# pragma once
-
-# include <openssl/macros.h>
-# if !OPENSSL_API_3
-#  define HEADER_EVPERR_H
-# endif
```

Comment:
Why did you remove this?

Reply:
This happened during `make update`, I didn't manually touch this file.

Reply:
Put back.

Reply:
Hmmm...  @mspncp, you might want to have a look at what happens with a make update.

Reply:
Sorry for the delay @levitte, I only noticed your message today. Too much input currently. I will take a look as soon as possible.

Reply:
> Hmmm... @mspncp, you might want to have a look at what happens with a make update.

@levitte the fix is straighforward and for you it's probably a five minute task: The mkerr.pl script needs to be adjusted to add the legacy guard, but not unconditionally; only if the file already has a legacy guard. I've written down the details in #10569. With my limited perl skills , it would take me significantly longer to figure it out.